core/src/com/mygdx/crazybomber/Room.java

```diff
@@ -0,0 +1,145 @@
+//TODO: when player enters, avatar is red for not ready, if blank, leave blank
+package com.mygdx.crazybomber;
+import com.badlogic.gdx.Game;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.Screen;
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.badlogic.gdx.scenes.scene2d.InputEvent;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ClickListener;
+
+public class Room implements Screen {
+    private Sprite background,avatar, avatarReady, avatarNotReady;
+    private SpriteBatch batch;
+    private Stage stage;
+    private Table table;
+    private TextButton buttonBack, buttonReady, buttonStart;
+    private Label heading;
+    private Skin skin;
+    private TextureAtlas atlas;
+    private Texture splashTexture;
+    private MapRepository mapRepository;
+    private StateRepository stateRepository;
+    private boolean ready = false;
+
+    public Room (Texture texture, final StateRepository stateRepository, MapRepository mapRepository) {
+        this.mapRepository = mapRepository;
+        this.splashTexture = texture;
+        this.stateRepository = stateRepository;
+    }
+
+    @Override
+    public void show() {
+        batch = new SpriteBatch();
+        background = new Sprite(splashTexture);
+        background.setSize(Gdx.graphics.getWidth(), Gdx.graphics.getHeight());
+
+        avatarReady = new Sprite(new Texture("ready.png"));
```

Comment:
instead of having different sprites, can we change a color somewhere from Green -> Red and vice versa?

Reply:
I was just testing this and I wanna make the ready to be actual different sprites instead of just green/red. but if I don't then yea ill do this

Reply:
Why diff sprites? whatd u find in ur tests

Reply:
I just tested to see if it worked at all. and different sprites because it's spicier (?)